src/main/app/fileUpload/documentManager.ts

```diff
@@ -12,6 +15,7 @@ export const uploadDocument = async (formData, s2sToken, req) => {
     authorization: `Bearer ${req.session.user.accessToken}`,
     serviceAuthorization: s2sToken,
   };
+  logger.info("Document Upload: Upload start");
```

Comment:
Better to convert this to debug or remove it as we want to reduce the loggers to save project cost.

Reply:
We need to have these logs at least temporarily so that platops can help use with the document upload problems. Can convert them to debug logs.